test/deserialization.test.ts

```diff
@@ -0,0 +1,40 @@
+import { getNodeClient } from './testHelpers';
+import { deserializeContractState } from '../src/deserialization';
+import { Buffer } from 'buffer/';
+import * as fs from 'fs';
+
+const client = getNodeClient();
+
+test('Deserialize state (two-step-transfer)', async () => {
+    const blockHash =
+        'fad0981b0424c6e1af746a39667628861481ac225f90decd233980311c2e19cb';
+    const contractAddress = { index: BigInt(1646), subindex: BigInt(0) };
+
+    const instanceInfo = await client.getInstanceInfo(
+        blockHash,
+        contractAddress
+    );
+    if (!instanceInfo) {
+        throw new Error(
+            'The instance info should exist for the provided block hash.'
+        );
+    }
+    const schema = Buffer.from(
+        fs.readFileSync('./test/resources/two-step-transfer-schema.bin')
```

Comment:
Could we add a test/example of how to use the embedded schema?

Reply:
It seems we can't, because the embedded schema comes from getModuleSource, which we haven't implemented yet (I just realise now). :sweat_smile:

Reply:
That is part of blocktech PR right?